package.json

```diff
@@ -11,12 +11,12 @@
   "scripts": {
     "clean": "rimraf dist lib",
     "build": "npm run clean && tsc",
-    "build-web": "webpack && webpack --env.production",
-    "build-node": "webpack --env.node && webpack --env.node --env.production",
```

Comment:
As far as I could tell, we don't use these scripts, but let me know if we still need them.